app/models/storage_location.rb

```diff
@@ -211,7 +217,13 @@ def self.csv_export_headers
   end
 
   def csv_export_attributes
-    [name, address, square_footage, warehouse_type, size]
+    attributes = [name, address, square_footage, warehouse_type, total_active_inventory_count]
+    inventory_items.sort{ |a, b| 
+      Item.find(a.item_id).name <=> Item.find(b.item_id).name
+    }.each { |item| 
+      attributes << item.quantity if Item.find(item.item_id).active
+    }
```

Comment:
Ditto here. Seems like you can just make use of the previously mentioned method to get these item names.